clients/web/src/stylesheets/body/plugins.styl

```diff
@@ -1,5 +1,7 @@
 @import 'nib'
 
+$warnColor = #f0ad4e
+
```

Comment:
Should we use this for the ```.g-plugin-warning``` below as well? I'm fine whichever one you think looks better, but it probably makes sense to only have one warn color (per file at least :P).